apollo-runtime/api/apollo-runtime.api

```diff
@@ -273,9 +290,14 @@ public final class com/apollographql/apollo3/network/http/HttpInfo$Key : com/apo
 }
 
 public abstract interface class com/apollographql/apollo3/network/http/HttpInterceptor {
+	public abstract fun dispose ()V
 	public abstract fun intercept (Lcom/apollographql/apollo3/api/http/HttpRequest;Lcom/apollographql/apollo3/network/http/HttpInterceptorChain;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 }
 
+public final class com/apollographql/apollo3/network/http/HttpInterceptor$DefaultImpls {
+	public static fun dispose (Lcom/apollographql/apollo3/network/http/HttpInterceptor;)V
```

Comment:
I'm not a huge fan of `DefaultImpls` but yea, having to implement `dispose()` all the time isn't great either so 👍 . Apollo 4 will be able to use JVM default methods 😅